### DISCUSSION TITLE
The team Id

### DISCUSSION THREAD

**Discussion by jacobxy** [saul/demofile#213] [Questions] [answered]
Is there a number for each team such as fnatic ?
Can I get the number of the team from the dem file with this project?

**Comment by pedrofornaza** [comment]
Hello!

Im not pretty sure about what you need. You can have the team clan names, flag and logo images and their respective players. Can you explain better to us?

**Comment by saul** [comment]
@jacobxy yeah I'm not sure exactly what you're referring to here - what is a team number?

**Comment by jacobxy** [comment]
How can I get the infomation such as "the team clan names, flag and logo images and their respective players. "?
I want to know which team is in the game !

**Comment by saul** [comment, marked as answer]
You can get the teams by doing:

```js
var terrorists = demo.teams[2];
terrorists.clanName
terrorists.flagImage
terrorists.logoImage
```
